### PR TITLE
Adding home page

### PR DESCRIPTION
**Pull request by bipin7yadav**

### CODE REVIEW

**Review by Swapnilkota108 — COMMENTED** [review]
I suggest to use an external css file

File: index.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bipin's Component Library</title>
+
+    <style>
```
Comment (srishti-maurya) [inline index.html]: you use external CSS it would be better for future

File: index.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+        .icons{
+            width: 20px;
+            border-radius: 4px;
+            background-color: snow;
+    
+        }
+
+        nav{
+            padding: 10px;
```
Comment (srishti-maurya) [inline index.html]: Suggestion : You can use rems or ems for better responsiveness

File: index.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            display: flex;
+            background-color: rgb(60, 171, 226);
```
Comment (srishti-maurya) [inline index.html]: you can define color variables may be

**Review by srishti-maurya — APPROVED** [review]
You can also add a preview link for convenience.

File: index.html (hunk 1 above)
Comment (bipin7yadav) [reply index.html]: Sure.

**Comment by bipin7yadav** [comment]
made changes such as added external css

**Pull request merged**